src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -111,7 +111,7 @@
       <TestCopyLocal Include="$(BUILDTOOLS_OVERRIDE_RUNTIME)/*.*" />
     </ItemGroup>
```

Comment:
Addressed in most recent push of changes.  This works for most, but not all tests (3 have some failures.  At least two (system.globalization*) are known already.